app/konnect/configure/runtime-manager/runtime-instances/gateway-runtime-docker.md

```diff
@@ -34,20 +32,16 @@ runtime instances.
 
 ### Run the quick setup script
 
-1. From the left navigation menu, open **Runtimes**.
+1. From the left navigation menu, open
+![runtimes icon](/assets/images/icons/konnect/icn-runtimes.svg){:.inline .konnect-icn .no-image-expand}
```

Comment:
Would it be useful to have a `{% konnect_icon 'icn-runtimes' %}` helper to render these?

Reply:
It certainly wouldn't hurt, if you're offering 😄